packages/grafana-data/src/transformations/index.ts

```diff
@@ -11,6 +11,6 @@ export {
 } from './standardTransformersRegistry';
 export { RegexpOrNamesMatcherOptions, ByNamesMatcherOptions, ByNamesMatcherMode } from './matchers/nameMatcher';
 export { RenameByRegexTransformerOptions } from './transformers/renameByRegex';
-export { outerJoinDataFrames } from './transformers/joinDataFrames';
+export { outerJoinDataFrames, join as joinAlignedData } from './transformers/joinDataFrames';
```

Comment:
^^^  we don't need to export this in `@grafana/data`, we can just import directly in core

I don't think we want to encourage people to use this function directly